examples/seismic/tutorials/07_elastic.ipynb

```diff
@@ -0,0 +1,409 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "class DGaussSource(WaveletSource):\n",
+    "    \n",
+    "    def wavelet(self, f0, t):\n",
+    "        a = 0.004\n",
+    "        return -2.*a*(t - 1/f0) * np.exp(-a * (t - 1/f0)**2)\n",
+    "\n",
+    "# Timestep size from Eq. 7 with V_p=6000. and dx=100\n",
+    "t0, tn = 0., 600.\n",
+    "dt = (10. / np.sqrt(2.)) / 6.\n",
+    "time_range = TimeAxis(start=t0, stop=tn, step=dt)\n",
+    "\n",
+    "src = RickerSource(name='src', grid=grid, f0=0.01, time_range=time_range)\n",
+    "src.coordinates.data[:] = [1000., 1000.]\n",
+    "src.show()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Now we create the velocity and pressure fields\n",
+    "vx= TimeFunction(name='vx', grid=grid, staggered=(0, 1, 0))\n",
+    "vz = TimeFunction(name='vz', grid=grid, staggered=(0, 0, 1))\n",
+    "txx = TimeFunction(name='txx', grid=grid)\n",
+    "tzz = TimeFunction(name='tzz', grid=grid)\n",
+    "txz = TimeFunction(name='txz', grid=grid, staggered=(0, 1, 1))\n",
+    "\n",
+    "# And finally we create the discretized/indexed symbols\n",
+    "VX = vx.indexed\n",
+    "VZ = vz.indexed\n",
+    "TXX = txx.indexed\n",
+    "TZZ = tzz.indexed\n",
+    "TXZ = txz.indexed"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def derivative(f, dim, order, stagger=centered):\n",
```

Comment:
same, we don't need this anymore, do we?

Reply:
same